src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -452,6 +452,21 @@ module Types = struct
             ~resolve:
               (fun _ {Mina_state.Blockchain_state.Poly.snarked_ledger_hash; _} ->
               Frozen_ledger_hash.to_string snarked_ledger_hash )
+        ; field "stagedLedgerProofEmitted" ~typ:(non_null bool)
+           ~doc: "block finished a staged ledger, and a proof was emitted from it and included into this block's proof"
+           ~args:Arg.[]
+           ~resolve:
+             (fun {ctx= coda; _}
+                  {Mina_state.Blockchain_state.Poly.state_hash; _} ->
+              let open Option.Let_syntax in
+              let%bind frontier =
+                Mina_lib.transition_frontier coda |> Pipe_lib.Broadcast_pipe.Reader.peek
+              in
+              match Transition_frontier.find frontier state_hash with
+              | None -> 
+                  false
```

Comment:
Should we throw an error here since this should never happen?

Reply:
I think we should return None (similar to what we do if the frontier is missing I think?) ANd then just mention in the docs "if the frontier is missing or the block gets removed from the frontier at just this moment we'll return null"